mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -40,7 +40,8 @@ use crate::s3_crt_client::put_object::S3PutObjectRequest;
 macro_rules! request_span {
     ($self:expr, $method:expr) => {{
         let counter = $self.next_request_counter();
-        tracing::debug_span!($method, id = counter)
+        // Request failures are emitted at warning verbosity, so emit their spans there too
+        tracing::warn_span!($method, id = counter)
```

Comment:
We are referring to the CRT here?

```suggestion
        // Request failures are emitted at warning verbosity by the CRT, so emit their spans as warning too
        tracing::warn_span!($method, id = counter)
```

Reply:
No, referring to the `warn!` messages we emit in this file. But this is actually wrong -- I always get `span` levels confused. It's the _lowest_ level at which the span should be emitted, and we want this span emitted in `debug!` messages, so the old code is correct.

Reply:
Gotcha! I was wondering the same but couldn't pin it down.